.github/workflows/awstest.yml

```diff
@@ -27,5 +26,5 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{secrets.AWS_KEY_ID}}
           AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_KEY_SECRET}}
           TOWER_ACCESS_TOKEN: ${{secrets.TOWER_ACCESS_TOKEN}}
-        run: |
+        run: | # Submits job to AWS batch, using the pre-define job definition. Setting JVM options to "-Xms1g -Xmx4g -XX:+UseG1GC" to allow staging of larger files.

```

Comment:
```suggestion
        run: | # Submits job to AWS batch, using a 'nextflow-big' instance. Setting JVM options to "-XX:+UseG1GC" for more efficient garbage collection when staging remote files.

```

Reply:
We may not need to specify `-Xms1g -Xmx4g` on the larger instance :) I ran a test for that too 🤞